staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4157,6 +4157,9 @@ const (
 	// Local ephemeral storage, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
 	// The resource name for ResourceEphemeralStorage is alpha and it can change across releases.
 	ResourceEphemeralStorage ResourceName = "ephemeral-storage"
+	// Process IDs, in count
```

Comment:
I don't think we want to add ResourcePID as a first-class resource.  We don't expect users to add requests/limits for pids, and we probably don't want pids to show up in the capacity/allocatable of the node.

Reply:
Is it OK for me to use a fixed string somewhere for this, or do I really need to keep this completely special cased?

Reply:
It definitely makes it rather messy.